src/lando/tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management import call_command
+
+import pytest
+
+
+@pytest.fixture
+def lando_version():
+    # The version file may or may not exist in the testing environment.
+    # We'll explicitly generate it to ensure it's there.
+    call_command('generate_version_file')
+
+    try:
+        from lando.version import version
+        return version
+    except ImportError:
+        pytest.fail("ImportError: Unable to import the version file after generation.")
```

Comment:
Ideally only the import stays in the try/except block. I also don't think we necessarily need to be calling `pytest.fail` here. An import error will cause a test to fail as an error, and I think it is self-evident in this case.

```suggestion
    from lando.version import version
    return version
```

Reply:
Very well.